### ISSUE TITLE
[BUG]  Model name "not found" 

### ISSUE THREAD

**Issue by mattehicks** [RupertAvery/DiffusionToolkit#139]
Model hash and name is shown in the generation parameters, but below on the model name line I get "not found".
I did a model data rebuild, but problem is still there.

![image](https://github.com/RupertAvery/DiffusionToolkit/assets/859222/7789855b-63f3-4350-a658-e8549b0ef1aa)
 
Version:  1.2.1

**Comment by acorderob** [comment]
Yep. This also happens with v1.3. I made sure the cache json file had the hash saved. I'm using SD.Next.

> Not found (94f45bf623)

```json
    "checkpoint/realcartoonRealistic_v8.safetensors": {
      "mtime": 1694898626.5027108,
      "sha256": "94f45bf6232fc2bb44b106daae23a911c452b8613dae3c3e6c51458cae0a5486"
    },
```

**Comment by RupertAvery** [comment]
This may be the case if the ModelRoot Path is not set.  I was primarily using the models to get the (original) hash, then supplementing with cache.json for the SHA256 hash.

I've changed it so that one is not dependent on the other.

**Comment by acorderob** [comment]
But i do have the model root set (the cache.json you see is the actual file and is symlinked in the SD folder).

![image](https://github.com/RupertAvery/DiffusionToolkit/assets/6293935/a003b2ad-432b-461e-a6d7-3c3b98873cdb)

**Comment by acorderob** [comment]
I think the fix in 1.4 is not quite right. Now sometimes i get the model repeated:

![image](https://github.com/RupertAvery/DiffusionToolkit/assets/6293935/6f91c8c1-aec5-4d71-9b4d-0012653cc49f)

Two of those copies are probably because the model is repeated in my cache.json file.

**Comment by RupertAvery** [comment]
yeah, there are more sources I'm pulling from, and sometimes they could be different files so I haven't really bothered to deal with them just yet.  There still maybe some other bugs lurking around.

Do you also use the CivitAI models info feature?

**Comment by acorderob** [comment]
I don't think i have used the "Download Civitai models" option.

**Comment by RupertAvery** [comment]
Just wondering if this is still an issue with the latest build

**Comment by acorderob** [comment]
In v1.5 i don't see *any* model name, even after downloading civitai models. The model is in models.json with the correct hash, so the only thing i can think of is that it is doing a case sensitive check on the hash (json has it in uppercase but hash is shown in lowercase in the interface).

**Comment by RupertAvery** [comment]
Sorry,  yes 1.5 had a bug where the model name was missing. It's fixed now.

**Comment by RupertAvery** [comment]
I'm using a caseless match on model hashes when loading the model names. Can you give me an image and a copy of your cache json? It would help identifying the issue.

**Comment by acorderob** [comment]
v1.5.1 fixed the model name for me.
